### PR TITLE
Add unit test for uncovered regions

### PR DESCRIPTION
**Pull request by CXWorks**
Hi,

Thanks for your time & patience to review this PR.

We are researchers focusing on Rust unit tests. By examine the existing code, we found a unit test can be added to improve the repo's overall unit test coverage(this project is already been well tested).
We only manually changed our assert statements to make it consistent with existing unit tests and the newly covered code region is:
https://github.com/mgeisler/textwrap/blob/872f221fede7aa78e730bb7895074455269a431a/src/refill.rs#L77-L80
Thanks again for reviewing.

### CODE REVIEW

**Comment by CXWorks** [comment]
Thanks for running the CI check, it seems one of them failed due to nightly toolchain issue, is there anything I can do?

File: src/refill.rs [hunk 1]
```diff
@@ -223,6 +223,14 @@ mod tests {
         assert_eq!(options.line_ending, LineEnding::LF);
     }
 
+    #[test]
+    fn test_unfill_missing() {
+        let (text, options) = unfill("í\n*\n/");
```
Comment (mgeisler) [inline src/refill.rs:228]: Could you simplify the test a bit? The `í` could probably be a `foo` and the final `/` could probably be something else, right? Basically, take the fuzzer output and turn it into something a human would write :smile:

File: src/refill.rs [hunk 2]
```diff
@@ -223,6 +223,14 @@ mod tests {
         assert_eq!(options.line_ending, LineEnding::LF);
     }
 
+    #[test]
+    fn test_unfill_missing() {
```
Comment (mgeisler) [inline src/refill.rs:227]: The word `missing` is confusing here. I think the test is covering the case where a bullet point list is _not_  actually seen as a list?

**Comment by mgeisler** [comment]
> Thanks for running the CI check, it seems one of them failed due to nightly toolchain issue, is there anything I can do?

I restarted the jobs — you could have done the same by pushing a different commit to the branch.

The PR looks good, great job at finding a test case to improve the coverage like this. I left two small comments about making the test case more human-friendly.

File: src/refill.rs (hunk 1 above)
Comment (CXWorks) [reply src/refill.rs:228]: Hhh, thanks for your comment, I will manually change it for this one and try to add this feature.

File: src/refill.rs (hunk 2 above)
Comment (CXWorks) [reply src/refill.rs:227]: I think you are right and I will manually change the name, thanks.

**Review by mgeisler — APPROVED** [review]
Awesome, thanks so much!

File: src/refill.rs (hunk 2 above)
Comment (mgeisler) [reply src/refill.rs]: Thanks!

**Comment by CXWorks** [comment]
Thanks for your feedback & comments, it's really helpful.

**Pull request merged**